include/ensmallen_bits/cmaes/cmaes.hpp

```diff
@@ -152,18 +201,55 @@ class CMAES
   double tolerance;
 
   //! The selection policy used to calculate the objective.
-  SelectionPolicyType selectionPolicy;
+  SelectionPolicyType selectionPolicy; 
+
+  // The weight initialization policy
+  WeightPolicyType weightPolicy;
+
+  // The update policy 
+  UpdatePolicyType updatePolicy;
+  
+ private:
+  size_t mu; // number of candidate solutions used to update the distribution parameters.
```

Comment:
Can you go over the comments and make sure they follow the style guidelines, see https://github.com/mlpack/mlpack/wiki/DesignGuidelines#comments for more details.